### PR TITLE
website/docs: add new section about impersonation

### PR DESCRIPTION
**Pull request by tanberry**
This PR adds a new section about Impersonation.

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

**Comment by rissson** [comment]
We also need to add a section about requiring a reason, I'll do that

File: website/docs/users-sources/user/user_basic_operations.md [hunk 1]
```diff
@@ -120,3 +120,18 @@ You may instead deactivate the account to preserve identity data.
...
+
+With authentik, an Admin can impersonate a user, meaning that the Admin temporarily assumes the identity of the user.
+
+1. In the Admin interface, navigate to **Directory > Users** to display all users.
+2. Click the name of the user to display the full User details page.
+3. On the Overview tab, beneath **User Details**, in the **Actions** area, click **Impersonate**.
+4. At the prompt, provide a reason why you are impersonating this user, and then click **Submit**.
+
+:::info
+An Admin can globally enable or disable impersonation in the [System Settings](../../sys-mgmt/settings.md#impersonation). By default, this option is set to true, meaning all users can be personified.
```
Comment (rissson) [inline website/docs/users-sources/user/user_basic_operations.md:134]: Not sure `personified` is the correct word here

Comment (tanberry) [reply website/docs/users-sources/user/user_basic_operations.md:134]: Ah, yeah, we need to use the word "impersonated". Thanks for catching that!

**Review by rissson — APPROVED** [review]

**Pull request merged**